### DISCUSSION TITLE
Overlapping nodes in GitHub Actions job graph

### DISCUSSION THREAD

**Discussion by coreycoto** [community/community#55411] [Actions]
### Select Topic Area

Bug

### Body

<img width="1345" alt="Screenshot 2023-05-13 at 1 38 07 PM" src="https://github.com/community/community/assets/200561/593e29f6-4746-4c9f-a7ed-f1156e667fde">

The job graph incorrectly overlays paths over paths that have no interdependencies.

**Comment by chrisrosner** [comment]
This bug caused me to spend a few minutes verifying that I had correctly defined my dependencies. I had. Without clicking on things in the graph, it falsely gives you the impression that the dependency tree is incorrect.

**Comment by pbonner-1** [comment]
We have the same problem with our workflow where the graph edges run underneath nodes despite there being a pretty obvious alternative arrangement that avoids this. This overlapping makes it look like there are dependencies between jobs until you highlight them which can be misleading if you're not aware of this problem.

**Comment by SamG101-Developer** [comment]
Hi did anyone else message about this privately / any progress going on in the background do you know? Thanks

**Reply by SamG101-Developer** [reply]
https://support.github.com/ticket/personal/0/4692267

(idk if that link is publicly accessible, but i raised a support ticket and they've offered to raise w product team on it)
